website/blog/2022-02-3-7.17.0.md

```diff
@@ -0,0 +1,110 @@
+
+## RegExp set notation and properties of strings ([#14125](https://github.com/babel/babel/pull/14125))
+
+Babel now supports the [RegExp set notation and properties of strings](https://github.com/tc39/proposal-regexp-set-notation/) proposal, which introduces three new regular expression features behind a new `v` flag:
+- Extended set notation, which allows computing difference, intersection and union of characters or strings sets
+  ```javascript
+  /[\p{Decimal_Number}--[0-9]]/v; // Non-ASCII decimal digits
+  /[\p{ASCII}&&\p{Letter}]/v; // ASCII letters
+  /[[\p{ASCII}&&\p{Letter}]\p{Number}]/v; // ASCII letters, or any digit
+  ```
+- Properties of strings, which allow using `\p` escapes with multi-codepoint properties
+  ```javascript
+  "Did you see the 👩🏿‍❤️‍💋‍👩🏾 emoji?".match(/\p{RGI_Emoji}/v). // ["👩🏿‍❤️‍💋‍👩🏾"]
+  ```
+- Multi-codepoint strings in sets, using a new `\q` escape
+  ```javascript
+  /[\r\n\q{\r\n|NEWLINE}]/v; // Matches \r, \n, \r\n or NEWLINE
+  ```
+
+You can enable this proposal using the `@babel/plugin-proposal-unicode-sets-regex` plugin:
+```json
+{
+  "plugins": ["@babel/plugin-proposal-unicode-sets-regex"]
+}
+```
+
+Babel relies on different third-party packages to transform regular expressions: we are grateful to the maintainers of [regjsparser](https://github.com/jviereck/regjsparser/), [regjsgen](http://github.com/benjamn/regjsgen) and [regexpu-core](https://github.com/mathiasbynens/regexpu-core/) who took time to review our pull requests!
+
+### `@@` and `^^` topic tokens for Hack-style pipes ([#13973](https://github.com/babel/babel/pull/13973))
+
+We are continuing to help the proposal authors evaluating different tokens for the [pipeline operator](https://github.com/tc39/proposal-pipeline-operator) proposal: one of the champions, [J. S. Choi](https://jschoi.org/), added support for two new tokens that you can try with the `topicToken` option:
+
+```json
+{
+  "plugins": [
+    ["@babel/plugin-proposal-pipeline-operator", {
+      "proposal": "hack",
+      "topicToken": "^^" // or "@@"
+    }]
+  ]
+}
+```
+
+### Experimental `@babel/register` rewrite ([#13782](https://github.com/babel/babel/pull/13782))
+
+We plan to release Babel 8 as an ESM package. However, this poses a big problem: how to load and run it synchronously in CommonJS packages?
+
+Usually most Babel consumers can run Babel asynchronously, or at least _load_ it asynchronously. This isn't true for `@babel/eslint-parser` and `@babel/register`, where we need to move parsing and transforming to a separate worker.
+
+Babel 7.16.0 exposes the new experimental `@babel/register` implementation at `@babel/register/experimental-worker`: it internally runs Babel asynchronously, so it's compatible with [`.mjs` configuration files](https://babeljs.io/docs/en/config-files#configuration-file-types) and with `.mjs` Babel plugins. It will be enabled by default in Babel 8, and you can already use it as a replacement of `@babel/register` with a few caveats:
```

Comment:
```suggestion
Babel 7.16.0 exposes the new experimental `@babel/register` implementation at `@babel/register/experimental-worker`: it internally runs Babel asynchronously, so it's compatible with [`.mjs` configuration files](https://babeljs.io/docs/en/config-files#configuration-file-types) and with `.mjs` Babel plugins. It will be enabled by default in Babel 8, and you can already use it as a replacement for `@babel/register` with a few caveats:
```